com.fafarunner.FaFaRunner.yml

```
@@ -0,0 +1,61 @@
+id: com.fafarunner.FaFaRunner
+
+# https://docs.flatpak.org/en/latest/available-runtimes.html
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```

```suggestion
runtime: org.freedesktop.Platform
runtime-version: '24.08'
```

obviously the comment is unecessary since it was not the latest.